p2p/test/p2ptest.go

```diff
@@ -13,20 +13,18 @@ import (
 	"github.com/qri-io/qri/repo/profile"
 	"github.com/qri-io/qri/repo/test"
 
-	net "gx/ipfs/QmPjvxTpVH8qJyQDnxnsxF9kv9jezKD1kozz1hs3fCGsNh/go-libp2p-net"
 	ma "gx/ipfs/QmYmsdtJ3HsodkePE3eU3TsCaP2YvPZJ4LoXnNkDE5Tpt7/go-multiaddr"
 	pstore "gx/ipfs/QmZR2XWVVBCtbgBWnQhWk2xcQfaR3W8faQPriAiaaj7rsr/go-libp2p-peerstore"
-	peer "gx/ipfs/QmdVrMn1LhB4ybb8hMVaMLXnA8XRSewMnK6YqXKXoTcRvN/go-libp2p-peer"
+	host "gx/ipfs/Qmb8T6YBBsjYsVGfrihQLfCJveczZnneSBqBKkYEBWDjge/go-libp2p-host"
 )
 
 // TestablePeerNode is used by tests only. Implemented by QriNode
 type TestablePeerNode interface {
-	Keys() pstore.KeyBook
-	Addrs() pstore.AddrBook
-	HostNetwork() net.Network
+	// add host here?
+	Host() host.Host
 	SimplePeerInfo() pstore.PeerInfo
-	AddPeer(pstore.PeerInfo) error
-	Connect() error
+	UpgradeQriConnection(pstore.PeerInfo) error
```

Comment:
Any opposition to naming this `UpgradeToQriConnection`, adding "To" implies transitioning from one state to another. As written I think it's possible to get confused and think we're going `QriConnection -> QriConnection` instead of `BasicConnection -> QriConnection`

Reply:
that's so funny, I literally did that after pushing and was going to ask how you felt about it.

Yes I totally agree.